### ISSUE TITLE
timer - resource check not working with v6.2 for more than 15mins

### ISSUE THREAD

**Issue by gowrisankar22** [concourse/concourse#5728]
## Summary

> We updated the concourse version from `v6.0` to `v6.2`. After the update `timer` resource is broken in all the pipelines more than 15mins and It recovered after that.

```
run check step: run check step: check: Backend error: Exit status: 500, message: {"Type":"","Message":"exit status 2","Handle":"","ProcessID":"","Binary":""}
```

## Steps to reproduce

> just update to `v6.2`
> the problem?
> timer is broken more than 15 mins and after that, it recovered.

## Expected results
it shouldnot break the timer.

## Actual results
`timer` resource is broken in all the pipeline more than 15mins.
```
run check step: run check step: check: Backend error: Exit status: 500, message: {"Type":"","Message":"exit status 2","Handle":"","ProcessID":"","Binary":""}
```

## Triaging info

* Concourse version: 6.2
* Browser (if applicable): NA
* Did this used to work?  : yes

**Comment by clarafu** [comment]
The error seems to be coming from 

https://github.com/concourse/concourse/blob/9f3c8970479647764fa1285e4574b08536eb1cbd/atc/worker/transport/hijack_streamer.go#L108

Is there anything you can tell us about how to reproduce this? Did you see that it error only after you upgraded? Were your workers properly talking to the ATCs while you were seeing these errors? Did this problem only happen once or has it been happening multiple times? Is this a problem with the upgrade process rather than a bug introduced in 6.2?

I'd like to ask that next time you write an issue, I'd like you to try to give us more information in order to help us debug. It is really difficult for us to figure out what caused the error and how to fix it if we do not have all the information of the current state your deployment was in. Because there is no way of us having complete visibility of your deployment during the time that the bug happened, that is why we need an exact way to reproduce the issue so that we can figure out what exactly went wrong.

**Comment by gowrisankar22** [comment]
@clarafu sorry for not detailing it out. Reproducing the issue is very simple. 

- You should be there in concourse `v6.0` not sure about `v6.1`. I hope it will be the same there as well.
- Have some pipeline which should use time resource and it should have run 1-2 builds.
- Now update to `v6.2`. 
- After the update, you can see time resource would have broken more than 15mins. Then it will auto-recover.

Hope,  this steps helps to reproduce the issue.

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
